src/test/lib/parts_test.ts

```diff
@@ -571,7 +571,7 @@ suite('Parts', () => {
       let passiveCalled = false;
       let onceCalled = false;
 
-      const listener = (_e: Event) => {
+      const listener = () => {
```

Comment:
In some VMs it's faster to have an explicit parameter if a function is going to be called with an argument

Reply:
So you prefer to have the unused parameter there?

Reply:
I don't care about this kinda micro-perf optimization in a test, but I'd like our eslint config to allow unused methods, provided they're named like `_foo`.

Reply:
Done in the last commit!